int16 Speed
int16 Steer
int8 steerMode
int8 gearMode
